app/components/satis/menu/component_controller.js

```diff
@@ -50,6 +46,16 @@ export default class extends ApplicationController {
       this.submenuTarget.classList.remove("hidden")
       this.submenuTarget.setAttribute("data-show", "")
       this.popperInstance.update()
+      const tableWrap = this.element.closest(".table-wrp")
+      const popperHeight = this.popperInstance.state.elements.popper.clientHeight + 20
+      if (tableWrap) {
+        this._tableWrpHeight ||= tableWrap?.style.minHeight
+        tableWrap.style.minHeight = `${popperHeight}px`
+      }
+      const firstInputEle = this.popperInstance.state.elements.popper.querySelector('form input:not([class=hidden])')
```

Comment:
Use full names for variables (firstInputElement). 
Should we also look for selects/textarea?

Reply:
I approved the changes. we only now have a bug for the tables that are on the lower end of the page.. (e.g. last table on the sales order overview)
textarea indeed, i dont know if we use it to filter on anything though. we just put focus on it so we can start typing a query without having to actually move into the dropdown/textfield to start typing the query. it saves one click.

Do we use selects?

Reply:
nope selected value will be prepopulated in input area